Server/Game/tests.py

```diff
@@ -448,4 +448,314 @@ def test11_query_games_user_success(self):
+		team1 = self.helper_golden_path_set_team_units()
+		team2 = self.helper_golden_path_set_team_units()
+		game_users = self.channel.createUsersAndPlaceUnits(credentials1, team1, credentials2, team2)
+		self.assertTrue(len(game_users) == 2)
+		unit = Unit.objects.filter(game=game_users.first().game, x_pos=0, y_pos=0).first()	# Get flier in location 0,0
+		newX = 0
+		newY = 0
+		valid_wait_command = {"Command":"TA", "Action":"Wait", "Game":"vs. second_user #1", "Unit":unit.id, "X":newX,"Y":newY}
+
+		# Moving onto self
+		self_move_command = copy.deepcopy(valid_wait_command)
+		self.channel.send(json.dumps(self_move_command))
+		result = json.loads(self.channel.receive())
+		self.assertTrue(result["Success"])
+		unit = Unit.objects.filter(pk=unit.id).first()
+		self.assertEqual(unit.x_pos, newX)
+		self.assertEqual(unit.y_pos, newY)
+
+		endTestLog("test14_take_action_basic_success")
+
+	def test15_take_action_valid_move_success(self):
+		startTestLog("test15_take_action_valid_move_success")
+
+		# Setup command
+		credentials1 = {"username":"first_user","password":"12345","email":"p1@email.com"}
+		credentials2 = {"username":"second_user","password":"12345","email":"p2@email.com"}
+		team1 = self.helper_golden_path_set_team_units()
+		team2 = self.helper_golden_path_set_team_units()
+		game_users = self.channel.createUsersAndPlaceUnits(credentials1, team1, credentials2, team2)
+		self.assertTrue(len(game_users) == 2)
+		unit = Unit.objects.filter(game=game_users.first().game, x_pos=0, y_pos=0).first()	# Get flier in location 0,0
+		newX = 0
+		newY = 1
+		valid_wait_command = {"Command":"TA", "Action":"Wait", "Game":"vs. second_user #1", "Unit":unit.id, "X":newX,"Y":newY}
+
+		# Moving south one spot
+		valid_move_command = copy.deepcopy(valid_wait_command)
+		self.channel.send(json.dumps(valid_move_command))
+		result = json.loads(self.channel.receive())
+		self.assertTrue(result["Success"])
+		unit = Unit.objects.filter(pk=unit.id).first()
+		self.assertEqual(unit.x_pos, newX)
+		self.assertEqual(unit.y_pos, newY)
+
+		endTestLog("test15_take_action_valid_move_success")
+
+	def test16_take_action_valid_move_through_ally_success(self):
+		startTestLog("test16_take_action_valid_move_through_ally_success")
+
+		# Setup command
+		credentials1 = {"username":"first_user","password":"12345","email":"p1@email.com"}
+		credentials2 = {"username":"second_user","password":"12345","email":"p2@email.com"}
+		team1 = self.helper_golden_path_set_team_units()
+		team2 = self.helper_golden_path_set_team_units()
+		game_users = self.channel.createUsersAndPlaceUnits(credentials1, team1, credentials2, team2)
+		self.assertTrue(len(game_users) == 2)
+		unit = Unit.objects.filter(x_pos=0, y_pos=0).first()	# Get flier in location 0,0
+		newX = 8
+		newY = 0
+		valid_wait_command = {"Command":"TA", "Action":"Wait", "Game":"vs. second_user #1", "Unit":unit.id, "X":newX,"Y":newY}
+
+		# Moving south one spot
+		valid_move_command = copy.deepcopy(valid_wait_command)
```

Comment:
Comment likely copied from above?  Looks like you are not just moving south one spot here ;)